src/prelude/prim.mo

```diff
@@ -376,6 +377,84 @@ func stableMemoryStoreBlob(offset : Nat64, val :  Blob) : () =
 func stableVarQuery() : shared query () -> async {size : Nat64} =
   (prim "stableVarQuery" : () -> (shared query () -> async {size : Nat64})) () ;
 
+// stable regions
+
+// Returns the region assigned to the original stable memory interface.
+func stableMemoryRegion() : Region =
+  (prim "stableMemoryRegion" : () -> Region) ();
+
+func regionNew() : Region =
+  (prim "regionNew" : () -> Region) ();
```

Comment:
I wonder if we should make this return an option, not trap, so users can recover from alloc failure, as they can for Region.grow. Might be worth getting it right now.

I even recall @rvanasa  suggesting Region.grow should return an option instead of previous size/-1 to encourage checking, but I was trying to stick to the system API as much as possible (for better or worse, probably worse). I guess we could later add Region.tryGrow for that.

Reply:
Nvm